packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -69,26 +61,26 @@ describe('my-application', () => {
   - **Not recommended**:
     - `it('shortens the address when the user has a long address')`
     - `it('shows an error when the user is not logged in')`
-  - Use Enzyme's [shallow](https://airbnb.io/enzyme/docs/api/shallow.html) instead of [mount](https://airbnb.io/enzyme/docs/api/mount.html) when possible to test components.
-  - Always `unmount` components at the end of the test.
+  - Use RTL's [render](https://testing-library.com/docs/react-testing-library/api/#render) when testing components.
+  - If you're using Enzyme, use [shallow](https://airbnb.io/enzyme/docs/api/shallow.html) instead of [mount](https://airbnb.io/enzyme/docs/api/mount.html) when possible to test components.
+    - Always `unmount` components at the end of the test.
```

Comment:
I would put these under a subheading for this section about `Rendering components` along with examples of each. It'll be a cleaner separation from the previous points and give you more space for code blocks (less cluttered than if it were still in the list).

Reply:
- [x] Create new subsection with examples